views/forms/DropdownCheckboxGroup/index.jsx

```diff
@@ -1,89 +1,119 @@
 import React, { useState } from 'react'
-import { useWatch } from 'react-hook-form'
-import { KeyboardArrowDown } from '@mui/icons-material'
-import { Menu, ListItem, TextField, InputAdornment } from '@mui/material'
-import ControlledCheckbox from '../ControlledCheckbox'
+import { FormControl, OutlinedInput, InputLabel, Select, MenuItem, Box, Chip } from '@mui/material'
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
```

Comment:
can you explain this? Why are we multiplying the height by 4.5?

Reply:
229c299